Update enablemfa.md (#3204)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/mfa/main/sms/enablemfa.md

> 1. From the Admin Console, select **Security** from the main menu and then **Authenticators**.

Reviewer: ```suggestion
1. In the Admin Console, go to **Security** > **Authenticators**.
```

> 2. Click the **Add Authenticator** button, select **Phone** for SMS/Voice.

Reviewer: ```suggestion
2. Click **Add Authenticator**, and then click **Add** on the Phone tile.
```

> 3. Click the Add button associated with the service.

Reviewer: ```suggestion
3. Choose whether verification messages are sent through **Voice call**, **SMS**, or select both options.
4. Select when users can use the Phone authenticator:
    * **Authentication and password reset**
    * **Authentication (MFA/SSO)
    * **Password reset**
5. Click **Add**.
```

1. In the Admin Console, go to **Security** > **Authenticators**.
2. Click **Add Authenticator**, and then click **Add** on the Phone tile.
3. Choose whether verification messages are sent through **Voice call**, **SMS**, or select both options.
4. Select when users can use the Phone authenticator:
    * **Authentication and password reset**

>     * **Authentication (MFA/SSO)

Reviewer: ```suggestion
    * **Authentication** (MFA/SSO)
```
